pallets/vault-registry/src/tests.rs

```diff
@@ -129,6 +129,26 @@ fn create_sample_vault_and_issue_tokens(issue_tokens: u128) -> DefaultVaultId<Te
 	create_vault_and_issue_tokens(issue_tokens, DEFAULT_COLLATERAL, DEFAULT_ID)
 }
 
+#[test]
+fn should_set_punishment_delay() {
```

Comment:
Nit: I'd rename to `set_punishment_delay_works()`

Reply:
Done.